packages/fxa-payments-server/src/components/PaymentFormV2/index.tsx

```diff
@@ -164,8 +175,8 @@ export const PaymentForm = ({
           component={CardElement}
           name="creditCard"
           label="Your card"
-          style={stripeElementStyles}
           className="input-row input-row--xl"
+          options={STRIPE_ELEMENT_STYLES}
```

Comment:
card elements don't take a style prop, but an options prop that has styles as a field in it...
https://github.com/stripe/react-stripe-js/blob/master/examples/class-components/0-Card-Minimal.js#L44

Reply:
Oh, I think there's another problem here: While the new react-stripe-js elements take options.style, the old react-stripe-elements currently in use *do* take a style prop (and *not* options):

https://github.com/stripe/react-stripe-elements#using-individual-element-components

So, we might need to tweak this somehow to use the old style for the old components currently in use on the site - until/unless we launch with the new library.

Reply:
Actually, disregard that... these are only changes to PaymentFormV2 and don't affect the old PaymentForm, so they're already separated between old & new Stripe elemenets

Reply:
yeah i figured it was not worth the effort to sweat the current UI